### PR TITLE
release 1.14.1

### PR DESCRIPTION
**Pull request by medyagh**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,52 @@
 # Release Notes
 
+
+## Version 1.14.1 - 2020-10-23
+
+FeatureS:
```
Comment (tstromberg) [inline CHANGELOG.md:6]: lowercase s

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,52 @@
...
+Bug Fixes:
+
+* docker: When creating networks, use MTU of built-in bridge network [#9530](https://github.com/kubernetes/minikube/pull/9530)
+* multinode: ensure worker node join control plane on restart [#9476](https://github.com/kubernetes/minikube/pull/9476)
+* Fix "--native-ssh" flag for "minikube ssh" [#9417](https://github.com/kubernetes/minikube/pull/9417)
+
+Minor Improvements:
+
+* Omit error message if 100-crio-bridge.conf has already been disabled [#9505](https://github.com/kubernetes/minikube/pull/9505)
+* Fix parallels driver initialization [#9494](https://github.com/kubernetes/minikube/pull/9494)
```
Comment (tstromberg) [inline CHANGELOG.md:19]: Move to bugfixes.

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,5 +1,52 @@
...
+* avoid re-downloading hyperkit driver [#9365](https://github.com/kubernetes/minikube/pull/9365)
+* improve gcp-auth addon failure policy [#9408](https://github.com/kubernetes/minikube/pull/9408)
+* Bump version for ingress dns addon [#9435](https://github.com/kubernetes/minikube/pull/9435)
+* Added deprecation warning for --network-plugin=cni [#9368](https://github.com/kubernetes/minikube/pull/9368)
+* Use correct port for echoserver - fixes "Connection reset by peer" LoadBalancer issue [#9511](https://github.com/kubernetes/minikube/pull/9511)
```
Comment (tstromberg) [inline CHANGELOG.md:24]: Remove - this was a public docs change.

File: CHANGELOG.md [hunk 4]
```diff
@@ -1,5 +1,52 @@
...
+* Update warning message for local proxy. [#9490](https://github.com/kubernetes/minikube/pull/9490)
+* Combine pause status test into existing pause test [#9466](https://github.com/kubernetes/minikube/pull/9466)
```
Comment (tstromberg) [inline CHANGELOG.md:26]: Remove - this was a test only change.

**Review by tstromberg — CHANGES_REQUESTED** [review]

File: CHANGELOG.md (hunk 1 above)
Comment (sharifelgamal) [inline CHANGELOG.md:6]: fix typo: Features

File: CHANGELOG.md [hunk 5]
```diff
@@ -1,5 +1,52 @@
...
+- Medya Ghazizadeh
+- Mikhail Zholobov
+- Nick Kubala
+- Pablo Caderno
+- Predrag Rogic
+- Priya Modali
+- Priya Wadhwa
+- Sharif Elgamal
+- Thomas Stromberg
+- Thomas Strömberg
```
Comment (sharifelgamal) [inline CHANGELOG.md:47]: remove the duplicate entry here

**Review by tstromberg — APPROVED** [review]
Approved once comments are addressed.

File: CHANGELOG.md (hunk 4 above)
Comment (sharifelgamal) [inline CHANGELOG.md:26]: this probably doesn't need to be included in release notes

**Pull request merged**